src/Vendr.PaymentProviders.Opayo/OpayoSettings.cs

```diff
@@ -69,6 +69,9 @@ public static class Defaults
         [PaymentProviderSetting(Name = "Display order lines on Opayo", Description ="Send the order line details to be shown on the payment providers final stage", SortOrder = 5)]
         public bool DisplayOrderLines { get; set; }
 
+        [PaymentProviderSetting(Name = "Order Line Description Property Alias", Description ="Specify the alias of the order line property to be passed through to the item description field (if left blank, this defaults to Product Name (Product SKU)", SortOrder = 6)]
+        public string OrderLineDescriptionPropertyAlias { get; set; }
+
         [PaymentProviderSetting(Name = "Order property alias: Description", Description = "Order property alias containing the description to send to Opayo", SortOrder = 2)]
         public string OrderPropertyDescription { get; set; }
```

Comment:
I'm happy with this addition, the only thing really is a consistency thing. 

Both of these properties act as descriptions, one being the order the other being the order line. I'd just make the setting property names be consistent, so make the new property `OrderLinePropertyDescription` and then also change the name of the setting in the `PaymentProviderSettingsAttribute` to `Order line property alias: Description` so that they both display the same. Description could be made to be similar, but I'm not too fussed about that one.

Reply:
Good idea, consistency is always key 👍 I have made the required changes to the property and its attributes